experimental/dds/tree/api-extractor.json

```diff
@@ -1,4 +1,15 @@
+/**
```

Comment:
Added by WB so we stop breaking their doc generation.  (I believe it's harmless.  :-)